lib/Backend/GlobOpt.cpp

```diff
@@ -1060,6 +1060,9 @@ GlobOpt::MergePredBlocksValueMaps(BasicBlock *block)
         this->KillAllObjectTypes();
     }
 
+    this->blockData.capturedArgs = JitAnew(this->alloc, BVSparse<JitArenaAllocator>, this->alloc);
+    this->blockData.changedSyms = JitAnew(this->alloc, BVSparse<JitArenaAllocator>, this->alloc);
```

Comment:
InitDataBlockData is called when there is no predecessor. Still need to create new BV when have predecessor so InitDataBlockData doesn't meet the requirement.

---
In reply to: [68328885](https://github.com/Microsoft/ChakraCore/pull/1152#discussion_r68328885) [](ancestors = 68328885,68328456)